app/build.gradle

```diff
@@ -129,6 +130,7 @@ dependencies {
     implementation Deps.log_interceptor
     implementation Deps.gson_converter
     implementation Deps.jackson_kotlin
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.12.0")
```

Comment:
Could you please use Deps dependency URL here?

Reply:
Done ;). Thanks-